torch/_utils.py

```diff
@@ -1054,3 +1054,15 @@ def fire_callbacks(self, *args: P.args, **kwargs: P.kwargs) -> None:
     ("exceptions", "StandardError"): ("builtins", "Exception"),
     ("UserDict", "UserDict"): ("collections", "UserDict"),
 }
+
+GPU_TYPES = ["cuda", "mps", "xpu"]
```

Comment:
Do we need to generalize `mps`?

Reply:
in latest code mps is  in the list of GPU_TYPES.